Resources/public/js/views/universaldiscovery/ez-universaldiscoverybrowseview.js

```diff
@@ -67,14 +67,18 @@ YUI.add('ez-universaldiscoverybrowseview', function (Y) {
         },
 
         /**
-         * `multipleChange` event handler. It sets the selected view
-         * `addConfirmButton` flag according to the new `multiple` attribute value.
+         * `multipleChange` and `isSelectableChange` events handler. It sets the selected view
+         * `addConfirmButton` flag according to the new `multiple` attribute value and passes
+         * new `isSelectable` function to the selected view.
          *
-         * @method _setSelectedViewConfirmButton
+         * @method _setSelectedViewAttrs
          * @protected
          */
-        _setSelectedViewConfirmButton: function () {
-            this.get('selectedView').set('addConfirmButton', this.get('multiple'));
+        _setSelectedViewAttrs: function () {
+            this.get('selectedView').setAttrs({
```

Comment:
I did it this way because `this.get('selectedView')` creates new instance of Y.eZ.UniversalDiscoverySelectedView so splitting it to separate event handlers will lead to losing `addConfirmButton` for selectedView if `isSelectable` changes and losing `isSelectable` for selectedView if `multiple` changes. So maybe there is better way of menaging selectedView?